crates/rspack_fs_node/src/node.rs

```diff
@@ -67,6 +67,7 @@ cfg_async! {
     pub write_file: JsFunction,
     pub mkdir: JsFunction,
     pub mkdirp: JsFunction,
+    pub rm: JsFunction,
```

Comment:
could you also change the `rm` here to `remove_dir_all`?

Reply:
done